code/game/atoms.dm

```diff
@@ -442,6 +442,9 @@ var/list/blood_splatter_icons = list()
 /atom/proc/ratvar_act()
 	return
 
+/atom/proc/rcd_act(mob/user, var/obj/item/weapon/rcd/the_rcd)

```

Comment:
only complaint that i can foresee is that this might be pathed too far up but im not sure if thats a real issue anyways